packages/gatsby-plugin-google-analytics/src/__tests__/gatsby-browser.js

```diff
@@ -29,12 +27,17 @@ describe(`gatsby-plugin-google-analytics`, () => {
         })
 
         beforeEach(() => {
+          jest.useFakeTimers()
           window.ga = jest.fn()
           window.requestAnimationFrame = jest.fn(cb => {
```

Comment:
with removing requestAnmationFrame from the code below you can remove this mock.